### PR TITLE
Add some lint errors to .pylintrc disable list (#2123)

### LINKED ISSUE #2123: lint CI error
Current lint target in CI fail: 
https://travis-ci.org/benoitc/gunicorn/jobs/590535074

```
$ python --version
188Python 3.7.1
189$ pip --version
190pip 19.0.3 from /home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/pip (python 3.7)
install
1911.38s$ pip install tox
21723.06s$ tox --recreate
218lint recreate: /home/travis/build/benoitc/gunicorn/.tox/lint
219lint installdeps: pylint
220lint develop-inst: /home/travis/build/benoitc/gunicorn
221lint installed: astroid==2.3.0,-e git+https://github.com/benoitc/gunicorn.git@bc6b2fd320bc70335a0f5597e86ec65429d015a0#egg=gunicorn,isort==4.3.21,lazy-object-proxy==1.4.2,mccabe==0.6.1,pylint==2.4.1,six==1.12.0,typed-ast==1.4.0,wrapt==1.11.2
222lint run-test-pre: PYTHONHASHSEED='284302499'
223lint run-test: commands[0] | pylint -j0 gunicorn tests/test_arbiter.py tests/test_config.py tests/test_http.py tests/test_invalid_requests.py tests/test_logger.py tests/test_pidfile.py tests/test_sock.py tests/test_ssl.py tests/test_statsd.py tests/test_systemd.py tests/test_util.py tests/test_valid_requests.py
224************* Module gunicorn.util
225gunicorn/util.py:326:8: W0127: Assigning the same variable 'module' to itself (self-assigning-variable)
226************* Module gunicorn.reloader
227gunicorn/reloader.py:50:16: R1724: Unnecessary "elif" after "continue" (no-else-continue)
228************* Module gunicorn.http.message
229gunicorn/http/message.py:240:12: R1723: Unnecessary "elif" after "break" (no-else-break)
230************* Module gunicorn.workers.gthread
231gunicorn/workers/gthread.py:154:12: R1723: Unnecessary "else" after "break" (no-else-break)
232
233-----------------------------------
234Your code has been rated at 9.99/10
235
236ERROR: InvocationError for command /home/travis/build/benoitc/gunicorn/.tox/lint/bin/pylint -j0 gunicorn tests/test_arbiter.py tests/test_config.py tests/test_http.py tests/test_invalid_requests.py tests/test_logger.py tests/test_pidfile.py tests/test_sock.py tests/test_ssl.py tests/test_statsd.py tests/test_systemd.py tests/test_util.py tests/test_valid_requests.py (exited with code 12)
237___________________________________ summary ____________________________________
238ERROR:   lint: commands failed
239The command "tox --recreate" exited with 1.
cache.2
240store build cache
252
253
254```

### PR DESCRIPTION
**Pull request by hramezani**
Fixes #2123

### CODE REVIEW

**Comment by hramezani** [comment]
@berkerpeksag I have added them to `.pylintrc` disable section.

**Comment by Djailla** [comment]
Except for `import-outside-toplevel` I find the warning pretty useful.

Especially for `module, xxx = module, yyy` which add no value here.

For the no else after break, it make the code more compact and avoid big if / else.

IMO only `import-outside-toplevel` should be in pylintrc

**Comment by hramezani** [comment]
@Djailla I agree with you.
@berkerpeksag What do you think?

**Comment by hramezani** [comment]
I added `import-outside-toplevel` to pylintrc disable list and fixed the rest of lint errors.

File: gunicorn/workers/gthread.py [hunk 1]
```diff
@@ -156,18 +156,17 @@ def murder_keepalived(self):
...
-                with self._lock:
-                    try:
-                        self.poller.unregister(conn.sock)
-                    except EnvironmentError as e:
-                        if e.errno != errno.EBADF:
-                            raise
-                    except KeyError:
-                        # already removed by the system, continue
-                        pass
+            self.nr_conns -= 1
```
Comment (tilgovi) [inline gunicorn/workers/gthread.py]: An empty line might make this more readable to me.

Comment (hramezani) [reply gunicorn/workers/gthread.py]: Done.

**Comment by benoitc** [comment]
I don't unrstand some changes in the reloader and gthread worker. how are they related? and what do they fix?

**Comment by hramezani** [comment]
@benoitc I can revert the change `gthread` and `reloader` modules and add `no-else-continue` and `no-else-break` to pylintrc disable list.
It is okay for you?

**Comment by tilgovi** [comment]
I'm neutral. I like using defaults for linters, rather than doing lots of customization. I also like some of these rules, some of the time.

For example, I really like when guards near the top of functions are under `if`, without an else:

``` python
def foo():
    if guard:
        return

    # no else because this is not about one thing _or_ the other, just about a precondition
```

It's rare that I find these lint rules to make the code worse, and if it does it can be locally disabled.

I'm 👍 for these changes the way they are in this PR right now.

**Comment by benoitc** [comment]
i'm agree with the general spirit. Although in this two cases, yes, I would prefer to keep it the current way. It's more readable. Rest of the patch is OK, thanks for this.

There is a bunch of refactoring that could be done later to modernize the current source but let's not all  the warnings dictate all of it  for now :)

**Comment by hramezani** [comment]
`pylint: disable=no-else-break` added to `workers/gthread.py`
`pylint: disable=no-else-continue` added to `reloader.py`

**Comment by benoitc** [comment]
thanks!

**Pull request merged**